jenkins/janitor.py

```diff
@@ -110,9 +110,11 @@ def clear_resources(project, col, resource):
             base.append('--%s=%s' % (resource.condition, col))
 
         print "Try to kill %s - %s" % (col, list(items))
```

Comment:
L110 uses ' and L112 uses ". Will you make this file consistent (all " or all ')?